Solutions/AristaAwakeSecurity/Analytic Rules/ModelMatchesWithMultipleDestinationsByDevice.yaml

```diff
@@ -0,0 +1,62 @@
+id: dfa3ec92-bdae-410f-b675-fe1814e4d43e
+name: Awake Security - Model Matches With Multiple Destinations By Device
```

Comment:
Can we make the name field shorter. Ideally it should be less than 50 characters.